### PR TITLE
chore(growth): Remove issue details mobile callout

### PR DESCRIPTION
**Pull request by PixelDust22**
image

### CODE REVIEW

File: static/app/components/suggestProjectCTA.tsx [hunk 1]
```diff
@@ -1,223 +0,0 @@
...
-      organizationId: organization.id,
-      feature: 'suggest_mobile_project',
-      status: 'dismissed',
-    });
-
-    this.setState({isDismissed: true});
-  };
-
-  openModal = () => {
-    trackAdvancedAnalyticsEvent('growth.opened_mobile_project_suggest_modal', {
```
Comment (scefali) [inline static/app/components/suggestProjectCTA.tsx:178]: @Neo-Zhixing we can remove the analytics events as well

Comment (Neo-Zhixing) [reply static/app/components/suggestProjectCTA.tsx:178]: good point

File: static/app/utils/analytics/growthAnalyticsEvents.tsx [hunk 2]
```diff
@@ -105,8 +104,6 @@ type GrowthAnalyticsKey = keyof GrowthEventParameters;
 export const growthEventMap: Record<GrowthAnalyticsKey, string> = {
   'growth.show_mobile_prompt_banner': 'Growth: Show Mobile Prompt Banner',
```
Comment (scefali) [inline static/app/utils/analytics/growthAnalyticsEvents.tsx:105]: @Neo-Zhixing you forgot about this one too

Comment (Neo-Zhixing) [reply static/app/utils/analytics/growthAnalyticsEvents.tsx]: removed this and another one. this should be all

**Review by scefali — APPROVED** [review]

**Pull request merged**